source/js/app/main.js

```diff
@@ -0,0 +1,7 @@
+// App entry point
```

Comment:
Can't comment it directly, but `.gitkeep` isn't needed as this file exists.

Reply:
`app` is an odd name; can we avoid placing output into the `source` directory? (Applies to the CSS too.)

Reply:
If not, both should probably be changed to having `src` and `dist` directories?

Reply:
😂 `.gitkeep` leftovers, good spot, cheers!

Reply:
The output within source dates back how the export process was initially set up for the sass. Agree it isn't ideal to have them nested in the build like this. I can take a look at splitting them out for the build, while keeping the directory structure of the exported files the same. (We could change the directory structure of the exported files too, but not in this PR.)